submissions/kovalov/dom/scripts/main.js

```diff
@@ -0,0 +1,30 @@
+import { getSelectedDataItem } from './data.js';
+import { createElement } from './element.js';
+
+const menuButton = document.querySelector('[data-menu-button]');
+const menuMobileContent = document.querySelector(
+  '[data-mobile-menu-content]'
+);
+const menuDesktopContent = document.querySelector(
+  '[data-desktop-menu-content]'
+);
+const mainContent = document.querySelector('[data-main-content]');
+
+menuButton.addEventListener('click', () => {
+  menuMobileContent.classList.toggle('opened');
+});
+
+menuMobileContent.addEventListener('click', appendContent);
+
+menuDesktopContent.addEventListener('click', appendContent);
+
+function appendContent(event) {
+  if (event.target.dataset.id) {
```

Comment:
You can improve your function reading by [this article](https://dev.to/dglsparsons/write-better-code-and-be-a-better-programmer-by-never-using-else-statements-4dbl)